Apps/Inc/UpdateDisplay.h

```diff
@@ -30,7 +30,8 @@ typedef enum{
 	UPDATEDISPLAY_ERR_FIFO_PUT		=-1,	// Error putting command in fifo
 	UPDATEDISPLAY_ERR_FIFO_POP		=-2,	// Error popping command from fifo
 	UPDATEDISPLAY_ERR_PARSE_COMP	=-3,	// Error parsing component/val in SetComponent
-	UPDATEDISPLAY_ERR_NO_CHANGE	=-4,		// No change in component value
+	UPDATEDISPLAY_ERR_NO_CHANGE	    =-4,	// No change in component value
+	UPDATEDISPLAY_ERR_DRIVER		=-5		// Driver call returned an error
 } UpdateDisplayError_t;
```

Comment:
Can we align these, just for niceness?

Reply:
I'll try, but it already looks aligned in my VS code, so I'm not sure why it's unaligned here.

Reply:
Ok, I think it's better now? At least it looks better [on the file on this latest commit](https://github.com/lhr-solar/Controls/pull/312/commits/2ca1f4ecc7f53f595055be2a1b064fc8eaeadcf5)